tools/cellxgene_census_builder/src/cellxgene_census_builder/build_soma/globals.py

```diff
@@ -13,7 +13,7 @@
 
 CENSUS_SCHEMA_VERSION = "2.0.1"
 
-CXG_SCHEMA_VERSION = "5.0.0"  # the CELLxGENE schema version supported
+CXG_SCHEMA_VERSION = "5.1.0"  # the CELLxGENE schema version supported
```

Comment:
@ebezzi @atolopko-czi - Should `CENSUS_SCHEMA_VERSION` also be bumped? If so, is it a `patch` or `minor` version bump?

Reply:
I would only bump if there are noticeable and traceable changes (e.g. major ontology changes, new/removed assays). We should be able to add some doc changes if we bump it.

Reply:
Yes we need to bump.the minor version number, and see my comment.
https://github.com/chanzuckerberg/cellxgene-census/issues/1194#issuecomment-2166838094